### PR TITLE
Allow update of open and close times

### PR DESCRIPTION
**Pull request by Skaty**

### CODE REVIEW

File: src/main/java/seedu/task/logic/commands/UpdateCommand.java [hunk 1]
```diff
@@ -31,26 +31,41 @@
...
-    private final DateTime openTime;
-    private final DateTime closeTime;
+    private final DateTime newOpenTime;
+    private final DateTime newCloseTime;
     private final Set<String> removedTags;
     private final UniqueTagList newTaskTags;
     
     public UpdateCommand(int targetIndex, String name, String openTime, String closeTime, Set<String> tagsToAdd, Set<String> tagsToRemove) throws IllegalValueException { 
         this.targetIndex = targetIndex;
+        System.out.println(name);
```
Comment (advaypal) [inline src/main/java/seedu/task/logic/commands/UpdateCommand.java]: Lol. Remove this

File: src/main/java/seedu/task/logic/parser/Parser.java [hunk 2]
```diff
@@ -33,8 +33,8 @@
     
     private static final Pattern UPDATE_TASK_DATA_ARGS_FORMAT = // '/' forward slashes are reserved for delimiter prefixes
             Pattern.compile("(?<name>[^/]*)"
-                    + "(s/(?<openTime>[^/]+))?"
-                    + "(c/(?<closeTime>[^/]+))?"
+                    + "( s/(?<openTime>[^/]+))?"
```
Comment (advaypal) [inline src/main/java/seedu/task/logic/parser/Parser.java:36]: Eh, I thought you asked me to remove the spaces?

**Review by advaypal — COMMENTED** [review]
See the comments I made

File: src/main/java/seedu/task/logic/commands/UpdateCommand.java (hunk 1 above)
Comment (Skaty) [reply src/main/java/seedu/task/logic/commands/UpdateCommand.java]: -.- oops

File: src/main/java/seedu/task/logic/parser/Parser.java (hunk 2 above)
Comment (Skaty) [reply src/main/java/seedu/task/logic/parser/Parser.java:36]: Oh, the spaces get regexed as name apparently and it screws up the name capturing.

**Review by advaypal — APPROVED** [review]

**Review by advaypal — APPROVED** [review]

**Pull request merged**